.gitmodules

```diff
@@ -4,3 +4,9 @@
 [submodule "enderboi/nixos"]
 	path = enderboi/nixos
 	url = git@github.com:kashifulhaque/nixos-config.git
```

Comment:
Why have you mentioned Enderboi's config location here ? 

Please remove this if you are not using it